ModulePresentationsForCAP/gap/ModulePresentationsForCAP.gi

```diff
@@ -1681,14 +1685,23 @@ InstallGlobalFunction( ADD_COEVALUATION_MORPHISM_RIGHT,
     
 end );
 
+##
+InstallGlobalFunction( ADD_EPIMORPHISM_FROM_SOME_PROJECTIVE_OBJECT, 
+    function( category )
+    
+    SetIsAbelianCategoryWithEnoughProjectives( category, true );
```

Comment:
It is better to set this property in the constructor of the category,
in order to trigger the derivations correctly.

Reply:
Done!